Dockerfile

```diff
@@ -13,15 +11,16 @@ ENV HOME /home/lounge
 ENV LOUNGE_HOME ${HOME}/data
 ENV LOUNGE_SRC ${HOME}/src
 
-RUN mkdir -p ${LOUNGE_SRC}
-WORKDIR ${LOUNGE_SRC}
-
-RUN curl -L https://github.com/thelounge/lounge/archive/v${LOUNGE_VERSION}.tar.gz | tar --strip-components 1 -xzf -
-
-RUN chown -R lounge:lounge ${LOUNGE_SRC}
+# Create a non-root user for lounge to run in.
+RUN useradd --create-home --uid 1000 lounge
```

Comment:
Linux systems tend to start assigning uids from `1000` to user, so this is just more explicit. I think explicitly declaring the uid makes it easier to figure out how to deal with volume mount permissions.